src/NServiceBus.WindowsPerformanceCounters/PerformanceCounters.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace NServiceBus
+{
+    public class PerformanceCounters
```

Comment:
In the core we tend to suffix classes like this with "Setting" of "Config" so perhaps rename to `PerformanceCounterConfig` ?